terra/executor/celery/executor.py

```diff
@@ -16,18 +16,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 from os import environ as env
-from concurrent.futures import Future, Executor, as_completed
+from concurrent.futures import as_completed
 from concurrent.futures._base import (RUNNING, FINISHED, CANCELLED,
                                       CANCELLED_AND_NOTIFIED)
 from threading import Lock, Thread
 import time
+from logging import NullHandler, StreamHandler
+from logging.handlers import SocketHandler
 
+from celery.signals import setup_logging
+
+from terra.executor.base import BaseFuture, BaseExecutor
+import terra
+from terra import settings
 from terra.logger import getLogger
 logger = getLogger(__name__)
 
 
-class CeleryExecutorFuture(Future):
+# stop celery from hijacking the logger
+@setup_logging.connect
+def setup_loggers(*args, **kwargs):
+  pass
```

Comment:
Is this leftover?

Reply:
I dont think so. That allows us to setup logging like we want